lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_generate_strings_file_from_code.rb

```diff
@@ -0,0 +1,93 @@
+module Fastlane
+  module Actions
+    class IosGenerateStringsFileFromCodeAction < Action
+      def self.run(params)
+        files = files_matching(paths: params[:paths], exclude: params[:exclude])
+        flags = [('-q' if params[:quiet]), ('-SwiftUI' if params[:swiftui])].compact
+        cmd = ['genstrings', '-o', params[:output_dir], *flags, *files]
+        out = Actions.sh_control_output(*cmd, print_command: FastlaneCore::Globals.verbose?, print_command_output: true)
+        out.scrub.strip.split("\n")
```

Comment:
This updated version is even better than the previous, IMHO. More compact and with no manual calls to `UI` because the `sh_control_output` and `genstrings` parameters combo does it all for us.

👏